DNN Platform/Library/Common/Globals.cs

```diff
@@ -3031,35 +3116,37 @@ public static string ResolveUrl(string url)
         /// <summary>
         /// Encodes the reserved characters.
         /// </summary>
-        /// <param name="QueryString">The query string.</param>
+        /// <param name="queryString">The query string.</param>
         /// <returns>Encoded content.</returns>
-        public static string EncodeReservedCharacters(string QueryString)
+        [Obsolete("Deprecated in v9.8.1, use System.Web.HttpUtility.UrlEncode instead, scheduled removal in v10.")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Let's recommend the .NET Standard version instead of the `System.Web` version
```suggestion
        [Obsolete("Deprecated in v9.8.1, use System.Net.WebUtility.UrlEncode instead, scheduled removal in v10.")]
```

Reply:
Addressed in latest commit, thanks.